src/lightning_app/utilities/proxies.py

```diff
@@ -191,35 +190,51 @@ def run(self):
                 self.list.append(1)
     """
 
-    def __init__(self, work: "LightningWork", delta_queue: "BaseQueue", interval: float = 5) -> None:
+    def __init__(self, work, delta_queue: "BaseQueue", interval: float = 0.1) -> None:
         super().__init__(daemon=True)
         self._work = work
         self._delta_queue = delta_queue
         self._interval = interval
         self._exit_event = Event()
         self._delta_memory = []
-        self._last_state = deepcopy(self._work.state)
```

Comment:
What do the changes in the state observer have to do with the state size? All it should care about is sending deltas and keeping its own copy of the state updated. Would you mind breaking this out into a separate PR?

Reply:
The StateObserver was wrongly implemented and sent duplicated deltas. I didn't manage to resolve this in an asynchronous manner, so I synchronized it with the main thread.

Reply:
Let me try to extract it to another PR.

Reply:
@awaelchli Done! I am still not 100 % it is working as expected, but the Flow is trying its best to reconcile errors.